swan-accpy/Dockerfile

```diff
@@ -0,0 +1,43 @@
+ARG VERSION_PARENT=v0.0.37
+
+FROM gitlab-registry.cern.ch/swan/docker-images/jupyter/swan-cern:${VERSION_PARENT}
+
+LABEL maintainer="swan-admins@cern.ch"
+ARG NB_UID="1000"
+ARG BUILD_TAG=daily
+ENV VERSION_DOCKER_IMAGE=$BUILD_TAG
+ENV ACCPY_PATH="/opt/acc-py"
```

Comment:
Does this variable need to be there when this image is instantiated at runtime or it is just a helper variable during the build process?

Reply:
I see, so you really need it to be present at runtime. Then all good.